mspray/apps/main/utils.py

```diff
@@ -975,6 +976,8 @@ def performance_report(spray_operator, queryset=None):
     """
     Update performance report for spray_operator.
     """
+
+    logger = logging.getLogger(__name__)
```

Comment:
I think the general convention is to do this at the top of the module/file.  One reason might be because it can then be used by everything else.

Also, do you think we need to have a test that ascertains we do this logging when we encounter that exception?

Reply:
cool, I put it within the function since I felt that was the only place where it was being used. let me move it to the top of the file